<!DOCTYPE html>
<html lang="en">
<!-- Template: https://github.com/luost26/academic-homepage -->
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <title>Publications - Zhaowei Yu</title>

    <!-- Stylesheets -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css" integrity="sha512-P5MgMn1jBN01asBgU0z60Qk4QxiXo86+wlFahKrsQf37c9cro517WzVSPPV1tDKzhku2iJ2FVgL67wG03SGnNA==" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.1/css/academicons.min.css" integrity="sha512-b1ASx0WHgVFL5ZQhTgiPWX+68KjS38Jk87jg7pe+qC7q9YkEtFq0z7xCglv7qGIs/68d3mAp+StfC8WKC5SSAg==" crossorigin="anonymous" />
    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css"> -->
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Raleway:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="/academic-homepage/assets/css/global.css">
</head>
<body class="bg-light" data-spy="scroll" data-target="#navbar-year" data-offset="100">
    <nav class="navbar navbar-expand-sm navbar-light bg-light fixed-top mb-5 shadow-sm">
    <div class="container">
        <a class="navbar-brand"><strong>Zhaowei Yu</strong></a>
        <button class="navbar-toggler" style="font-size: 1em; padding: 0.5em;" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <i class="fas fa-map"></i> Menu
        </button>

        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav ml-auto">
                
                <li class="nav-item ">
                    <a class="nav-link" href="/academic-homepage/">Home</a>
                </li>
                
                <li class="nav-item active">
                    <a class="nav-link" href="/academic-homepage/publications">Publications</a>
                </li>
                
            </ul>
        </div>
    </div>
</nav>

    <div class="container">
        

<div class="row">
    <div class="col-12 col-lg-10">
        
        
        <h2 class="pt-4" id="year-2024">2024</h2>
        <div class="my-0 p-0 bg-white shadow-sm rounded-sm">
            
                
<div class="d-none d-md-block">
    <div class="row no-gutters border-bottom border-gray"><div class="col-md-3 col-xl-2 mb-md-0 p-md-3">
            <img data-src="/academic-homepage/assets/images/covers/CondSig.png" alt="Decoding the genomic landscape of chromatin-associated biomolecular condensates" class="lazy w-100 rounded-sm" src="/academic-homepage/assets/images/empty_300x200.png">
        </div>
        <div class="col-md-9 col-xl-10 p-3 pl-md-0"><h5 class="mt-0 mb-1 font-weight-normal">Decoding the genomic landscape of chromatin-associated biomolecular condensates</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Wang Q*, </span><span class="text-body">
            Zhang Q*, </span><span class="text-body">
            Tian Y, </span><span class="text-body">
            Yan G, </span><span class="text-body">
            Zhu J, </span><span class="text-body">
            Zhu G, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Communications</strong></i> 2024 </p>
            <p class="mt-0 mb-0 small text-muted">In this study, we introduce our newly developed computational framework, CondSigDetector, to predict genome-wide landscapes of chromatin-associated biomolecular condensates.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2023.08.23.554542v1">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/CondSig">[Code]</a>
                
                
            </p>

        </div>
    </div>
</div>

<div class="card w-100 d-md-none border-top-0 border-right-0 border-left-0 rounded-0" style="overflow-y: scroll;"><img data-src="/academic-homepage/assets/images/covers/CondSig.png" alt="Decoding the genomic landscape of chromatin-associated biomolecular condensates" class="lazy card-img w-100" src="/academic-homepage/assets/images/empty_300x200.png"><div class="card-img-overlay d-flex align-items-start flex-column" style="background-color: rgba(255, 255, 255, 0.85);">
        <div class="mb-auto"></div>
        <div>
            <h5 class="mt-0 mb-1 font-weight-normal">Decoding the genomic landscape of chromatin-associated biomolecular condensates</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Wang Q*, </span><span class="text-body">
            Zhang Q*, </span><span class="text-body">
            Tian Y, </span><span class="text-body">
            Yan G, </span><span class="text-body">
            Zhu J, </span><span class="text-body">
            Zhu G, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Communications</strong></i> 2024 </p>
            <p class="mt-0 mb-0 small text-muted d-none d-sm-block">In this study, we introduce our newly developed computational framework, CondSigDetector, to predict genome-wide landscapes of chromatin-associated biomolecular condensates.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2023.08.23.554542v1">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/CondSig">[Code]</a>
                
                
            </p>
        </div>
    </div>
</div>
            
                
<div class="d-none d-md-block">
    <div class="row no-gutters border-bottom border-gray"><div class="col-md-3 col-xl-2 mb-md-0 p-md-3">
            <img data-src="/academic-homepage/assets/images/covers/PSPire.png" alt="Machine learning predictor PSPire screens for phase-separating proteins lacking intrinsically disordered regions" class="lazy w-100 rounded-sm" src="/academic-homepage/assets/images/empty_300x200.png">
        </div>
        <div class="col-md-9 col-xl-10 p-3 pl-md-0"><h5 class="mt-0 mb-1 font-weight-normal">Machine learning predictor PSPire screens for phase-separating proteins lacking intrinsically disordered regions</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Hou S, </span><span class="text-body">
            Hu J, </span><span class="text-body">
            <strong>Yu Z</strong>, </span><span class="text-body">
            Li D, </span><span class="text-body">
            Liu C, </span><span class="text-body">
            Zhang Y</span></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Communications</strong></i> 2024 </p>
            <p class="mt-0 mb-0 small text-muted">In this work, we introduce PSPire, a machine learning predictor that incorporates both residue-level and structure-level features for the precise prediction of PSPs.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.nature.com/articles/s41467-024-46445-y">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/PSPire">[Code]</a>
                
                
            </p>

        </div>
    </div>
</div>

<div class="card w-100 d-md-none border-top-0 border-right-0 border-left-0 rounded-0" style="overflow-y: scroll;"><img data-src="/academic-homepage/assets/images/covers/PSPire.png" alt="Machine learning predictor PSPire screens for phase-separating proteins lacking intrinsically disordered regions" class="lazy card-img w-100" src="/academic-homepage/assets/images/empty_300x200.png"><div class="card-img-overlay d-flex align-items-start flex-column" style="background-color: rgba(255, 255, 255, 0.85);">
        <div class="mb-auto"></div>
        <div>
            <h5 class="mt-0 mb-1 font-weight-normal">Machine learning predictor PSPire screens for phase-separating proteins lacking intrinsically disordered regions</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Hou S, </span><span class="text-body">
            Hu J, </span><span class="text-body">
            <strong>Yu Z</strong>, </span><span class="text-body">
            Li D, </span><span class="text-body">
            Liu C, </span><span class="text-body">
            Zhang Y</span></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Communications</strong></i> 2024 </p>
            <p class="mt-0 mb-0 small text-muted d-none d-sm-block">In this work, we introduce PSPire, a machine learning predictor that incorporates both residue-level and structure-level features for the precise prediction of PSPs.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.nature.com/articles/s41467-024-46445-y">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/PSPire">[Code]</a>
                
                
            </p>
        </div>
    </div>
</div>
            
        </div>
        
        
        <h2 class="pt-4" id="year-2022">2022</h2>
        <div class="my-0 p-0 bg-white shadow-sm rounded-sm">
            
                
<div class="d-none d-md-block">
    <div class="row no-gutters border-bottom border-gray"><div class="col-md-3 col-xl-2 mb-md-0 p-md-3">
            <img data-src="/academic-homepage/assets/images/covers/NCB2022.png" alt="Allele-specific H3K9me3 and DNA methylation co-marked CpG-rich regions serve as potential imprinting control regions in pre-implantation embryo." class="lazy w-100 rounded-sm" src="/academic-homepage/assets/images/empty_300x200.png">
        </div>
        <div class="col-md-9 col-xl-10 p-3 pl-md-0"><h5 class="mt-0 mb-1 font-weight-normal">Allele-specific H3K9me3 and DNA methylation co-marked CpG-rich regions serve as potential imprinting control regions in pre-implantation embryo.</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Yang H*, </span><span class="text-body">
            Bai D*, </span><span class="text-body">
            Li Y*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Wang C, </span><span class="text-body">
            Sheng Y, </span><span class="text-body">
            Liu W, </span><span class="text-body">
            Gao S, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Cell Biology</strong></i> 2022 </p>
            <p class="mt-0 mb-0 small text-muted">In this study, we comprehensively analyse the association between DNA methylation and H3K9me3 reprogramming in mouse pre-implantation embryos and reveal that CpG-rich genomic loci with high H3K9me3 signal and DNA methylation level (CHM) are hotspots of DNA methylation maintenance during pre-implantation embryogenesis. We further profile the allele-specific epigenetic map with unprecedented resolution in gynogenetic and androgenetic embryos, respectively, and identify 1,279 allele-specific CHMs, including 19 known imprinting control regions (ICRs).</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.nature.com/articles/s41556-022-00900-4">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/PCAR">[Code]</a>
                
                
            </p>

        </div>
    </div>
</div>

<div class="card w-100 d-md-none border-top-0 border-right-0 border-left-0 rounded-0" style="overflow-y: scroll;"><img data-src="/academic-homepage/assets/images/covers/NCB2022.png" alt="Allele-specific H3K9me3 and DNA methylation co-marked CpG-rich regions serve as potential imprinting control regions in pre-implantation embryo." class="lazy card-img w-100" src="/academic-homepage/assets/images/empty_300x200.png"><div class="card-img-overlay d-flex align-items-start flex-column" style="background-color: rgba(255, 255, 255, 0.85);">
        <div class="mb-auto"></div>
        <div>
            <h5 class="mt-0 mb-1 font-weight-normal">Allele-specific H3K9me3 and DNA methylation co-marked CpG-rich regions serve as potential imprinting control regions in pre-implantation embryo.</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Yang H*, </span><span class="text-body">
            Bai D*, </span><span class="text-body">
            Li Y*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Wang C, </span><span class="text-body">
            Sheng Y, </span><span class="text-body">
            Liu W, </span><span class="text-body">
            Gao S, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Nature Cell Biology</strong></i> 2022 </p>
            <p class="mt-0 mb-0 small text-muted d-none d-sm-block">In this study, we comprehensively analyse the association between DNA methylation and H3K9me3 reprogramming in mouse pre-implantation embryos and reveal that CpG-rich genomic loci with high H3K9me3 signal and DNA methylation level (CHM) are hotspots of DNA methylation maintenance during pre-implantation embryogenesis. We further profile the allele-specific epigenetic map with unprecedented resolution in gynogenetic and androgenetic embryos, respectively, and identify 1,279 allele-specific CHMs, including 19 known imprinting control regions (ICRs).</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.nature.com/articles/s41556-022-00900-4">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/PCAR">[Code]</a>
                
                
            </p>
        </div>
    </div>
</div>
            
                
<div class="d-none d-md-block">
    <div class="row no-gutters border-bottom border-gray"><div class="col-md-3 col-xl-2 mb-md-0 p-md-3">
            <img data-src="/academic-homepage/assets/images/covers/MC2022.png" alt="CpG island reconfiguration for the establishment and synchronization of polycomb functions upon exit from naive pluripotency. " class="lazy w-100 rounded-sm" src="/academic-homepage/assets/images/empty_300x200.png">
        </div>
        <div class="col-md-9 col-xl-10 p-3 pl-md-0"><h5 class="mt-0 mb-1 font-weight-normal">CpG island reconfiguration for the establishment and synchronization of polycomb functions upon exit from naive pluripotency. </h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Huo D*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Li R, </span><span class="text-body">
            Gong M, </span><span class="text-body">
            Sidoli S, </span><span class="text-body">
            Lu X, </span><span class="text-body">
            Hou Y, </span><span class="text-body">
            Dai Z, </span><span class="text-body">
            Kong Y, </span><span class="text-body">
            Liu G, </span><span class="text-body">
            Jensen O, </span><span class="text-body">
            Xie W, </span><span class="text-body">
            Helin K, </span><span class="text-body">
            Xiong C, </span><span class="text-body">
            Li G, </span><span class="text-body">
            Zhang Y, </span><span class="text-body">
            Wu X</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Molecular Cell</strong></i> 2022 </p>
            <p class="mt-0 mb-0 small text-muted">In this study, we show that the expression of the long KDM2B isoform (KDM2BLF), which contains the demethylase domain, is specifically induced at peri-implantation and that its H3K36me2 demethylase activity is required for PcG enrichment at CGIs. Moreover, KDM2BLF interacts with BRG1/BRM-associated factor (BAF) and stabilizes BAF occupancy at CGIs for subsequent gain of accessibility, which precedes PcG enrichment.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(22)00103-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276522001034%3Fshowall%3Dtrue">[Paper]</a>
                
                
            </p>

        </div>
    </div>
</div>

<div class="card w-100 d-md-none border-top-0 border-right-0 border-left-0 rounded-0" style="overflow-y: scroll;"><img data-src="/academic-homepage/assets/images/covers/MC2022.png" alt="CpG island reconfiguration for the establishment and synchronization of polycomb functions upon exit from naive pluripotency. " class="lazy card-img w-100" src="/academic-homepage/assets/images/empty_300x200.png"><div class="card-img-overlay d-flex align-items-start flex-column" style="background-color: rgba(255, 255, 255, 0.85);">
        <div class="mb-auto"></div>
        <div>
            <h5 class="mt-0 mb-1 font-weight-normal">CpG island reconfiguration for the establishment and synchronization of polycomb functions upon exit from naive pluripotency. </h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Huo D*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Li R, </span><span class="text-body">
            Gong M, </span><span class="text-body">
            Sidoli S, </span><span class="text-body">
            Lu X, </span><span class="text-body">
            Hou Y, </span><span class="text-body">
            Dai Z, </span><span class="text-body">
            Kong Y, </span><span class="text-body">
            Liu G, </span><span class="text-body">
            Jensen O, </span><span class="text-body">
            Xie W, </span><span class="text-body">
            Helin K, </span><span class="text-body">
            Xiong C, </span><span class="text-body">
            Li G, </span><span class="text-body">
            Zhang Y, </span><span class="text-body">
            Wu X</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Molecular Cell</strong></i> 2022 </p>
            <p class="mt-0 mb-0 small text-muted d-none d-sm-block">In this study, we show that the expression of the long KDM2B isoform (KDM2BLF), which contains the demethylase domain, is specifically induced at peri-implantation and that its H3K36me2 demethylase activity is required for PcG enrichment at CGIs. Moreover, KDM2BLF interacts with BRG1/BRM-associated factor (BAF) and stabilizes BAF occupancy at CGIs for subsequent gain of accessibility, which precedes PcG enrichment.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(22)00103-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276522001034%3Fshowall%3Dtrue">[Paper]</a>
                
                
            </p>
        </div>
    </div>
</div>
            
        </div>
        
        
        <h2 class="pt-4" id="year-2020">2020</h2>
        <div class="my-0 p-0 bg-white shadow-sm rounded-sm">
            
                
<div class="d-none d-md-block">
    <div class="row no-gutters border-bottom border-gray"><div class="col-md-3 col-xl-2 mb-md-0 p-md-3">
            <img data-src="/academic-homepage/assets/images/covers/ncHMR_detector.png" alt="ncHMR detector: a computational framework to systematically reveal non-classical functions of histone modification regulators." class="lazy w-100 rounded-sm" src="/academic-homepage/assets/images/empty_300x200.png">
        </div>
        <div class="col-md-9 col-xl-10 p-3 pl-md-0"><h5 class="mt-0 mb-1 font-weight-normal">ncHMR detector: a computational framework to systematically reveal non-classical functions of histone modification regulators.</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Hu S*, </span><span class="text-body">
            Huo D*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Chen Y*, </span><span class="text-body">
            Liu J, </span><span class="text-body">
            Liu L, </span><span class="text-body">
            Wu X, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Genome Biology</strong></i> 2020 </p>
            <p class="mt-0 mb-0 small text-muted">In this study, we develop ncHMR detector, the first computational framework to predict non-classical functions and cofactors of a given HMR, based on ChIP-seq data mining.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01953-0">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/ncHMR_detector">[Code]</a>
                
                
            </p>

        </div>
    </div>
</div>

<div class="card w-100 d-md-none border-top-0 border-right-0 border-left-0 rounded-0" style="overflow-y: scroll;"><img data-src="/academic-homepage/assets/images/covers/ncHMR_detector.png" alt="ncHMR detector: a computational framework to systematically reveal non-classical functions of histone modification regulators." class="lazy card-img w-100" src="/academic-homepage/assets/images/empty_300x200.png"><div class="card-img-overlay d-flex align-items-start flex-column" style="background-color: rgba(255, 255, 255, 0.85);">
        <div class="mb-auto"></div>
        <div>
            <h5 class="mt-0 mb-1 font-weight-normal">ncHMR detector: a computational framework to systematically reveal non-classical functions of histone modification regulators.</h5>
            <p class="mt-0 mb-0 small"><span class="text-body">
            Hu S*, </span><span class="text-body">
            Huo D*, </span><span class="text-body">
            <strong>Yu Z*</strong>, </span><span class="text-body">
            Chen Y*, </span><span class="text-body">
            Liu J, </span><span class="text-body">
            Liu L, </span><span class="text-body">
            Wu X, </span><span class="text-body">
            Zhang Y</span>
<mark>(* <i> co-first</i>)</mark></p>
            <p class="mt-0 mb-0 small"><i><strong>Genome Biology</strong></i> 2020 </p>
            <p class="mt-0 mb-0 small text-muted d-none d-sm-block">In this study, we develop ncHMR detector, the first computational framework to predict non-classical functions and cofactors of a given HMR, based on ChIP-seq data mining.</p>
            
            <p class="small pb-0 mb-0 lh-125 text-muted abstract-links">
                
                
                <a target="_blank" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01953-0">[Paper]</a>
                
                
                
                <a target="_blank" href="https://github.com/TongjiZhanglab/ncHMR_detector">[Code]</a>
                
                
            </p>
        </div>
    </div>
</div>
            
        </div>
        
    </div>

    <div class="col-2 d-none d-lg-block">
        <div id="navbar-year" class="nav nav-pills flex-column sticky-top" style="top: 80px">
            
            <a class="nav-link d-block" href="#year-2024">2024</a>
            
            <a class="nav-link d-block" href="#year-2022">2022</a>
            
            <a class="nav-link d-block" href="#year-2020">2020</a>
            
        </div>
    </div>

</div>

    </div>
    <footer class="footer">
    <div class="container">
        <div class="row my-3">
            <div class="col-6">
                <div class="text-muted">
                    <i>Last updated: Aug 2024</i>
                </div>
            </div>
            <div class="col-6">
                <div class="text-right text-muted">
                    Theme by jekyll-template from <a href="https://github.com/luost26/academic-homepage" target="_blank"><i class="fas fa-pencil-ruler"></i> academic-homepage</a>
                </div>
            </div>
        </div>
    </div>
</footer>


    <!-- Scripts -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.lazy/1.7.9/jquery.lazy.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/js/bootstrap.min.js" integrity="sha512-XKa9Hemdy1Ui3KSGgJdgMyYlUg1gM+QhL6cnlyTe2qzMCYm4nAZ1PsVerQzTTXzonUR+dmswHqgJPuwCq1MaAg==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/github-buttons/2.14.2/buttons.min.js" integrity="sha512-OYwZx04hKFeFNYrWxIyo3atgGpb+cxU0ENWBZs72X7T9U+NoHPM1ftUn/Mfw7dRDXrqWA6M1wBg6z6fGE32aeA==" crossorigin="anonymous"></script>
    <script src="/academic-homepage/assets/js/common.js"></script>
</body>
</html>
